File: 107598019/HW2/Software Architecture_XML/Software Architecture/usecase/addCourse/AddCourseInput.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Software_Architecture.usecase.addCourse
{
    public interface AddCourseInput
    {
        string getTitle();

        void setTitle(string title);

        string getDescription();

        void setDescription(string description);

        string getSuitable();

        void setSuitable(string suitable);

        string getPrice();

        void setPrice(string price);

        string getNotice();

        void setNotice(string notice);

        string getOther();

        void setOther(string other);
    }
}

File: 107598042/hw2/src/main/java/UseCase/AddCourse/AddCourseInput.java
package UseCase.AddCourse;


public class AddCourseInput {
    public String courseName;
    public String description;
    public String notes;
    public String remark;
    public String suitableObject;
    public double price;

    public AddCourseInput(String courseName, String description, String notes, String remark, String suitableObject, double price) {
        this.courseName = courseName;
        this.description = description;
        this.notes = notes;
        this.remark = remark;
        this.suitableObject = suitableObject;
        this.price = price;
    }
}
